app/components/treatment/Treatment.js

```diff
@@ -21,7 +21,15 @@ export class Treatment extends Component {
 
   render() {
 
-    const { treatment } = this.props
+    const { plan } = this.props
+    const { treatments } = plan
+    const treatmentId = Number(this.props.params.treatmentId)
```

Comment:
This is interesting, so if `react-router` the params actually get passed as a prop to the loaded component?